### DISCUSSION TITLE
best way to turn heater on/off?

### DISCUSSION THREAD

**Discussion by balerion** [wrongisthenewright/ebusd-configuration-ariston-bridgenet#11] [Q&A] [answered]
Hi, thanks for the wonderful work done.
I was wondering if people had a clever way to use the ebusd to turn the heater on/off based on different temperature readings.
The only way I have found so far is to change the z1 setpoint so it is higher/lower than the ariston thermostat temperature reading.
Is there a way to send a direct "boiler on" command? I don't think i could find this in the ariston.csv file.

**Comment by wrongisthenewright** [comment]
Hi, can you please add some details of your use case/scenario?
In general you can use the heating status command present in the CSV to turn on and off the heating system. If you have hybrid system this turn off/on both but leave active (if enabled) the DHW feature.
If you have a boiler only system you should check if the above command in the CSV is working with your specific device.

**Reply by balerion** [reply]
> Hi, can you please add some details of your use case/scenario?

Hi, yes sure! I have an Ariston One condensing boiler, and I have several thermometers around the house which I am using, in home assistant, to regulate the temperature in my home by turning the heating on/off. On the ebus there is also an Ariston Sensys thermostat, which has its own temperature reading. However its physical placement makes the temperature reading inaccurate, so i am ignoring this.

> In general you can use the heating status command

Thank you for your suggestion. I will try this, but if the Z1 set point is below the Z1 room_temp read by the Ariston thermostat, this will not turn the boiler on i think. Should I just have a very high set point?

I was also thinking along the lines of having a "z1_heat_request" but as far as i can understand this is a read only property. Is this correct? Perhaps one can use "z1_heat_request_mode"? i am unsure what this actually does tho.

**Reply by wrongisthenewright** [reply, marked as answer]
Hi, the question you're posing should be addressed by some professional. What I can suggest is:

1) read the boiler user manual (all pages :) )
2) read the boiler installaer manual, all pages also here, but focus on the electtric connections and PCB layout
3) install a smart switch on the TA1 connector of the boiler
4) program the boikler to use the TA1 port as a prioritary device, over the sensys
5) create the logic in HA (I think you should look at groups)

If in doubt ask Ariston Customer service, as far as my experience go they're not fast on reply but eventually they get back to you.

P.S:
the smart switch(point 3) can be any wireless smart switch that has volt free/dry contact output, eg sonoff mini. you can find the N,L connector nearby on the boiler PCB

**Reply by balerion** [reply]
Thanks! I will try and report

**Reply by wrongisthenewright** [reply]
Please take care when connecting the smart switch, it MUST be volt free/dry contact otherwise you'll fry your PCB
